assets/licenses.json

```diff
@@ -0,0 +1,19 @@
+{
+  "bloc_test": "The MIT License (MIT)\nCopyright (c) 2018 Felix Angelov\n\nPermission is hereby granted, free of charge, to any person\nobtaining a copy of this software and associated documentation\nfiles (the \"Software\"), to deal in the Software without restriction,\nincluding without limitation the rights to use, copy, modify, merge,\npublish, distribute, sublicense, and/or sell copies of the Software,\nand to permit persons to whom the Software is furnished to do so,\nsubject to the following conditions:\n\nThe above copyright notice and this permission notice shall be included\nin all copies or substantial portions of the Software.\n\nTHE SOFTWARE IS PROVIDED \"AS IS\", WITHOUT WARRANTY OF ANY KIND,\nEXPRESS OR IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF\nMERCHANTABILITY, FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT.\nIN NO EVENT SHALL THE AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM,\nDAMAGES OR OTHER LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR\nOTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE\nUSE OR OTHER DEALINGS IN THE SOFTWARE.",
```

Comment:
Could you also add the FlatIcon license? https://file000.flaticon.com/downloads/license/license.pdf

Reply:
> Not sure about what to do with this https://file000.flaticon.com/downloads/license/license.pdf
> Aren't we supposed to directly credit the author(s) of the used artworks instead of just adding a url to that license?
that's what i was wondering about

Reply:
hm right, maybe we make another change and add a Credits section for this license?